doc/source/tutorial/tutorial_IO.ipyml

```diff
@@ -555,75 +555,169 @@ cells:
 - markdown: |
     <div class="alert alert-warning">
       **Warning:** Currently, metadata associated with arrays cannot be saved and loaded when working with CSV and Excel files.
+      This restriction does not apply however to metadata associated with sessions.
     </div>
 
 
 - markdown: |
     ## Loading and Dumping Sessions
     
-    One of the main advantages of grouping arrays in session objects is that you can load and save all of them in one shot.
+    One of the main advantages of grouping arrays, axes and groups in session objects is that you can load and save all of them in one shot. Like arrays, it is possible to associate metadata to a session. These can be saved and loaded in all file formats. 
 
 
 - markdown: |
     ### Loading Sessions (CSV, Excel, HDF5)
+    
+    To load the items of a session, you have two options:
+    
+    1) Instantiate a new session and pass the path to the Excel/HDF5 file or to the directory containing CSV files to the Session constructor:
 
 
 - code: |
-
+    # create a new Session object and load all arrays, axes, groups and metadata 
+    # from all CSV files located in the passed directory
+    csv_dir = get_example_filepath('population_session')
+    session = Session(csv_dir)
+    
+    # create a new Session object and load all arrays, axes, groups and metadata
+    # stored in the passed Excel file
+    filepath_excel = get_example_filepath('population_session.xlsx')
+    session = Session(filepath_excel)
+    
+    # create a new Session object and load all arrays, axes, groups and metadata
+    # stored in the passed HDF5 file
+    filepath_hdf = get_example_filepath('population_session.h5')
+    session = Session(filepath_hdf)
+    
+    print(session.summary())
 
   id: 34
 
-- code: |
+- markdown: |
+    2) Call the ``load`` method on an existing session and pass the path to the Excel/HDF5 file or to the directory containing CSV files as first argument:
 
 
+- code: |
+    # create a session with two arrays
+    session = Session([('arr1', ndtest((3, 3))), ('arr2', ndtest((2, 2, 2)))])
```

Comment:
The idea is to show that the `load` method add new items to a session and does not delete the existing items.

Reply:
I would rather use an empty session (using Session()) and use .load several times on it IF THAT MAKES SENSE given the current files (if you have distinct arrays in different files), because this is closer to what people actually do. If that's not the case, leave the thing as it is.